docs/_static/index.html

```diff
@@ -0,0 +1,10 @@
+<!-- This page is created automatically by documentation builder -->
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Redirecting to main branch docs</title>
+    <meta charset="utf-8">
+    <meta http-equiv="refresh" content="0; url=./3.1/index.html">
+    <link rel="canonical" href="./3.1/index.html">
```

Comment:
Will this always redirect to 3.1?

Reply:
This is an autogenerated page, should not be there at all and it doesn't play any role in the multi version docs. Removed, added to .gitignore like in other projects.